test/unit/LiquidSpec.hs

```diff
@@ -111,10 +110,11 @@ spec = do
     it "runs the liquid haskell exe" $ do
       let
         fp = cwd </> "test/testdata/liquid/Evens.hs"
-        -- fp = "/home/alanz/tmp/haskell-proc-play/Evens.hs"
-        -- uri = filePathToUri fp
-      Just (ef, (msg:_)) <- runLiquidHaskell fp
-      msg `shouldSatisfy` isPrefixOf "RESULT\n[{\"start\":{\"line\":9,\"column\":1},\"stop\":{\"line\":9,\"column\":8},\"message\":\"Error: Liquid Type Mismatch\\n  Inferred type\\n    VV : {v : Int | v == (7 : int)}\\n \\n  not a subtype of Required type\\n    VV : {VV : Int | VV mod 2 == 0}\\n"
-      ef `shouldBe` ExitFailure 1
-
+      result <- runLiquidHaskell fp
+      case result of
+        Just (ef, msg:_) -> do
+          msg `shouldSatisfy` isPrefixOf "RESULT\n[{\"start\":{\"line\":9,\"column\":1},\"stop\":{\"line\":9,\"column\":8},\"message\":\"Error: Liquid Type Mismatch\\n  Inferred type\\n    VV : {v : Int | v == (7 : int)}\\n \\n  not a subtype of Required type\\n    VV : {VV : Int | VV mod 2 == 0}\\n"
+          ef `shouldBe` ExitFailure 1
+        Just (_, []) -> error "Didn't get any output from liquid binary"
+        Nothing -> error "This test requires 'liquid' binary to be present on your PATH. Follow the steps in docs/Hacking.md to install it."
```

Comment:
Attempt to give more informative error than "failed pattern match" when liquid is not installed.

Reply:
I did something similar [here](https://github.com/haskell/haskell-ide-engine/pull/900/files#diff-454abe77fa4d590ec22dd7ca842a9246R28)

Reply:
Ok, thanks. I'm going to remove the changes to that file from my PR..